spec/support/features/session_helper.rb

```diff
@@ -35,6 +35,16 @@ def sign_up
       user
     end
 
+    def sign_up_with_sp(loa: 'loa1')
+      allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
+      user = sign_up_and_set_password
+      fill_in 'Phone', with: '202-555-1212'
+      send("#{loa}_sp_session")
+      select_sms_delivery
+
+      user
+    end
```

Comment:
I did that originally, reek complains that the method is being controlled by a boolean/parameter. From what I could tell, the solution is to break it up into multiple methods